DuckDuckGo.xcodeproj/project.pbxproj

```diff
@@ -10043,8 +10047,8 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/DuckDuckGo/BrowserServicesKit";
 			requirement = {
-				kind = exactVersion;
-				version = 122.1.0;
+				branch = "sam/vpn-ui-improvements";
+				kind = branch;
```

Comment:
This is the only PR of the batch that needs a BSK update, I'll assign you a separate task for this on Wednesday.